types/busboy-body-parser/busboy-body-parser-tests.ts

```diff
@@ -0,0 +1,32 @@
+import express = require("express");
+import busboyBodyParser = require("./index");
+
+const app = express();
+
+//test types
+app.use("/form1", busboyBodyParser(), (req, res, next) => {
+    //test multi
+    let multiFile = (<busboyBodyParser.File[]>(<busboyBodyParser.Files>req.files)["a"])[0];
+    console.log(<Buffer>multiFile.data);
```

Comment:
What is the point of these type assertions? Did you mean to [just use `// $ExpectType Buffer`](https://github.com/DefinitelyTyped/DefinitelyTyped/search?q=expecttype&unscoped_q=expecttype)

Reply:
Yes I think it's the same thing . Just with the cast, my IDE tell me if the type is correct or not .